client/src/components/Home/SectionTwo.jsx

```diff
@@ -233,15 +249,24 @@
                       {">"}
                     </span>
                   </label>
+                  
                   <input
-                    name="name"
-                    type="text"
-                    placeholder="your name"
-                    className="input input-bordered w-full max-w-xs"
-                    onChange={handleInputChange}
-                    value={formData.name}
-                  />
-                  <label className="select-text cursor-text label">
+  name="name"
+  type="text"
+  placeholder="Your Name"
+  className="input input-bordered w-full max-w-xs"
+  onChange={handleInputChange}
+  onFocus={(event) => {
+    if (event.target.value === formData.name) {
+      event.target.value = "";
+      handleInputChange(event);
+    }
+  }}
```

Comment:
> The `onFocus` event handler for the "name" input field directly manipulates the event target's value, which differs from the approach used for "phoneNumber" and "bio" fields where `clearDefaultText` is utilized. For consistency and maintainability, consider using the `clearDefaultText` function for all input fields, ensuring a uniform approach to clearing default text.

It did not work for that input field so that's why i tried another approach

Reply:
<input
  name="name"
  type="text"
  placeholder="Your Name"
  className="input input-bordered w-full max-w-xs"
  onChange={handleInputChange}
  onClick={(event) => clearDefaultText(event, 'name')}
  value={formData.name}
/>
this is what i used and when clicking on the input field it didn't erase what was written

Reply:
i fixed it